app/views/payments/new.html.slim

```diff
@@ -0,0 +1,68 @@
+.forms-container
+  .grid-container
+    .payment-form.grid-x
+      .medium-6.medium-centered.cell
+        form[
+          id="payment-form"
+          target="_blank"
+          action="https://<-- seu servico -->"
```

Comment:
Here you can already use `subscriptions_path` (which, with a POST request will go to `subscriptions#create`) as `action`